content/tutorials/webrtc/datachannels/en/index.html

```diff
@@ -180,46 +179,46 @@ <h2 id="just-show-me-the-action">Configuring data channels</h2>
 };
 </pre>
 
-<p>It is also possible to add a <code>maxRetransmits</code> option (the number of times to try before failing) but you can only specify maxRetransmits or maxPacketLifeTime, not both. For UDP semantics, set <code>maxRetransmits</code> to 0 and <code>ordered</code> to <code>false</code>. For more information, see <a href="http://tools.ietf.org/html/rfc4960" title="IETF RFC 4960">RFC 4960</a> (SCTP) and <a href="http://tools.ietf.org/html/rfc3758" title="IETF RFC 3758">RFC 3758</a> (SCTP partial reliability).</p>
+<p>It is also possible to add a <code>maxRetransmits</code> option (the number of times to try before failing), but you can only specify maxRetransmits or maxPacketLifeTime, not both. For UDP semantics, set <code>maxRetransmits</code> to <code>0</code> and <code>ordered</code> to <code>false</code>. For more information, see <a href="https://tools.ietf.org/html/rfc4960" title="IETF RFC 4960">RFC 4960 - SCTP</a> and <a href="https://tools.ietf.org/html/rfc3758" title="IETF RFC 3758">RFC 3758 - SCTP Partial Reliability Extension</a>.</p>
```

Comment:
I think it might be better just to put the names in full and remove the RFC numbers, something like this:

```
For more information, see the following IETF RFCs: <a href="https://tools.ietf.org/html/rfc4960" title="IETF RFC 4960">Stream Control Transmission Protocol</a> and <a href="https://tools.ietf.org/html/rfc3758" title="IETF RFC 3758">Stream Control Transmission Protocol Partial Reliability Extension</a>.
```

Reply:
Will fix